disperser/cmd/apiserver/main.go

```diff
@@ -114,6 +115,14 @@ func RunDisperserServer(ctx *cli.Context) error {
 		ratelimiter = ratelimit.NewRateLimiter(reg, globalParams, bucketStore, logger)
 	}
 
+	if config.MaxBlobSize < 0 || config.MaxBlobSize > 64*1024*1024 {
```

Comment:
Should it fail if config.MaxBlobSize == 0?

Reply:
yeah, I will exclude 0